reference-architectures/reference-architecture-ccs.md

```diff
@@ -0,0 +1,157 @@
+
+Security and Compliance Center: This platform offers a comprehensive suite of tools to assess, monitor, and maintain the security and compliance of your cloud environment. It provides insights and controls to help organizations meet regulatory requirements, adhere to best practices, and protect against threats.
+
+IBM Cloud Security and Compliance Center Workload Protection: This service offers functionality to protect workloads, get deep cloud and container visibility, posture management (compliance, benchmarks, CIEM), vulnerability scanning, forensics, and threat detection and blocking.
+
+This reference architecture showcases how these services form a foundational security layer that enhances data protection, simplifies compliance, and strengthens overall cloud security for any workload in IBM Cloud. 
+
+## Architecture diagram
+{: #architecture-diagram}
+
+The below diagram represents the architecture for Core Security Services on IBM cloud and reuses the [best practices](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-about) for IBM Cloud for Financial Services.
+
+![Architecture.](core-security-services-architecture.svg "Architecture"){: caption="Figure 1. Reference Architecture" caption-side="bottom"}
+
+The architecture is anchored by three fundamental services: Key Protect, Secrets Manager, and IBM Cloud Security Services and Workload Protection. These services provide integration endpoints for any customer workload hosted on IBM Cloud.
+
+1. Key Protect
+
+Key Protect is responsible for centrally managing the lifecycle of encryption keys used by IBM Cloud Object Storage (COS) buckets, Secrets Manager, and event notification resources. Additionally, it can manage encryption keys for any customer workload requiring protection.
+
+2. Secrets Manager
+
+Secrets Manager securely stores and manages sensitive information, including API keys, credentials, and certificates. It utilizes encryption keys from Key Protect to encrypt sensitive data and to seal/unseal vaults holding the secrets. It is preconfigured to send events to the Event Notifications service, allowing customers to set up email or SMS notifications. Moreover, it is automatically configured to forward all API logs to the customer's logging instance.
+
+3. Security Compliance Center
+
+The Security Compliance Center instance is preconfigured to scan all resources provisioned by the reference architecture. It can be expanded to accommodate the unique workloads of customers.
+
+
+Cloud Object Storage (COS) buckets are set up to receive logs from Logging and Alerting Services. Each bucket is configured to encrypt data at rest using encryption keys managed by Key Protect.
```

Comment:
```suggestion
IBM Cloud Object Storage buckets are set up to receive logs from Logging and Alerting Services. Each bucket is configured to encrypt data at rest by using encryption keys managed by Key Protect.
```

Logging and alerting are not Cloud services. Should this be
IBM Cloud Log Analysis and ? ?